QA_CUSTOM_NETWORK.md

```diff
@@ -1,80 +1,135 @@
 # Custom Network
 
 1. **Settings screen**
-    1. Go to Settings
-    1. Check you see "General Settings" as the title for the first collection of settings
-    1. Check you see "Developer Settings" as the title for the second collection of settings
+    1. [ ] Go to Settings
+    1. [ ] Check you see "General Settings" as the title for the first collection of settings
+    1. [ ] Check you see "Developer Settings" as the title for the second collection of settings
 
 1. **Risk Disclaimer screen**
-    1. Go to Network Settings
-    1. Ensure you see the `RISK DISCLAIMER` screen
-    1. Verify that there is a disclaimer text on a yellow background
-    1. Confirm there is a button at the bottom of the page to acknowledge the disclaimer
+    1. [ ] Go to Network Settings
+    1. [ ] Ensure you see the `RISK DISCLAIMER` screen
+    1. [ ] Verify that there is a disclaimer text on a yellow background
+    1. [ ] Confirm there is a button at the bottom of the page to acknowledge the disclaimer
 
 1. **Network Pre-Settings Screen**
-    1. Click on "I UNDERSTAND"
-    1. Check you see the `NETWORK PRE-SETTINGS` screen
-    1. Check you see two pre-settings options: Mainnet and Testnet
-    1. Check you see a button at the bottom of the page to customize the network
+    1. [ ] Click on "I UNDERSTAND"
+    1. [ ] Check you see the `NETWORK PRE-SETTINGS` screen
+    1. [ ] Check you see two pre-settings options: Mainnet and Testnet
+    1. [ ] Check you see a button at the bottom of the page to customize the network
 
 1. **CUSTOM NETWORK SETTINGS Screen**
-    1. Click on "CUSTOMIZE"
-    1. Check you see the `CUSTOM NETWORK SETTINGS` screen
-    1. Check you see a warning message text in a yellow background
-    1. Check you see a form with the following values:
+    1. [ ] Click on "CUSTOMIZE"
+    1. [ ] Check you see the `CUSTOM NETWORK SETTINGS` screen
+    1. [ ] Check you see a warning message text in a yellow background
+    1. [ ] Check you see a form with the following values:
         - Node URL
         - Explorer URL
         - Explorer Service URL
+        - Transaction Mining Service URL
         - Wallet Service URL (optional)
         - Wallet Service WS URL (optional)
-    1. Check you see a button at the bottom of the page to send the form
+
+        >[!INFO]
+        >Wallet Service fields only appears if the wallet is allowed on Wallet Service feature.
```

Comment:
Oh! My bad. I made this markup using the Obsidian syntax. I did it because I copy the content in this document to an obsidian file, and I manage the QA from there. This way I can go ticking the checkbox and adding notes.